dashboard/app/models/reference_guide.rb

```diff
@@ -62,6 +62,21 @@ def remove_serialization
     File.delete(file_path)
   end
 
+  def copy_to_course_version(course_version)
```

Comment:
nit: name this `destination_course_version` or something? This might just be me but I get confused when a parameter is named the same as a class method